### DISCUSSION TITLE
DAOIP-4: Proposal Types

### DISCUSSION THREAD

**Discussion by thelastjosh** [metagov/daostar#50] [General]
A thread to discuss suggestions, critiques, and improvements to [DAOIP-4: Proposal Types](https://github.com/metagov/daostar/blob/main/DAOIPs/daoip-4.md).

**Comment by 0xPabloLI** [comment]
I am frequently using this document to structure my proposal. What I think is probably most missing is a more generic template for small governance change (not specified to on-chain). For example, [this proposal](https://forum.plancker.org/t/recommended-standards-for-planckerdao-voting-sbt-allocation/53) that I recently made in the PlankcerDAO.
